README.md

```diff
@@ -76,10 +76,8 @@ Jessie deliberately does not include:
 * ternary expressions (`?`)
 
 ## Future Tasks
-
-1. Better error messages: The default error messages aren't as clear as they could be. 
-2. More written tests for each of the rules
-3. Enforce more of the Jessie subset through an ESLint plugin that creates new rules, including the rules below:
+1. Add more extensive tests (currently, only the most obvious usages are tested in many cases.)
+2. Enforce more of the Jessie subset through an ESLint plugin that creates new rules, including the rules below:
```

Comment:
```suggestion
1. Enforce more of the Jessie subset through an ESLint plugin that creates new rules, including the rules below:
```

Reply:
Markdown renumbers this for you anyway, so I prefer to just "1." everything, so no one reading the source imagines that they see accurate numbers.

Reply:
Good idea!